quickstart/prerequisites.md

```diff
@@ -8,29 +8,272 @@ All worker nodes must satisfy the following requirements:
+
+   - ServiceMonitor from `monitoring.coreos.com` API group
+
+   - Ingress from `networking.k8s.io` API group and from extensions API group
+
+   - Route from `route.openshift.io` API group
+
+   - All resources from `jaegertracing.io` API group
+
+   
+
+
+{% tab title="Sample `ClusterRole` YAML " %}
+
+```text
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Release.Name }}-service-account
+  namespace: {{ .Release.Namespace }}
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: mayastor-cluster-role
+rules:
+- apiGroups: ["apiextensions.k8s.io"]
+  resources: ["customresourcedefinitions"]
+  verbs: ["create", "list"]
+  # must read diskpool info
+- apiGroups: ["datacore.com"]
+  resources: ["diskpools"]
+  verbs: ["get", "list", "watch", "update", "replace", "patch"]
+  # must update diskpool status
+- apiGroups: ["datacore.com"]
+  resources: ["diskpools/status"]
+  verbs: ["update", "patch"]
+  # external provisioner & attacher
+- apiGroups: [""]
+  resources: ["persistentvolumes"]
+  verbs: ["get", "list", "watch", "update", "create", "delete", "patch"]
+- apiGroups: [""]
+  resources: ["nodes"]
+  verbs: ["get", "list", "watch"]
+
+  # external provisioner
+- apiGroups: [""]
+  resources: ["persistentvolumeclaims"]
+  verbs: ["get", "list", "watch", "update"]
+- apiGroups: ["storage.k8s.io"]
+  resources: ["storageclasses"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: [""]
+  resources: ["events"]
+  verbs: ["list", "watch", "create", "update", "patch"]
+- apiGroups: ["snapshot.storage.k8s.io"]
+  resources: ["volumesnapshots"]
+  verbs: ["get", "list"]
+- apiGroups: ["snapshot.storage.k8s.io"]
+  resources: ["volumesnapshotcontents"]
+  verbs: ["get", "list"]
+- apiGroups: [""]
+  resources: ["nodes"]
+  verbs: ["get", "list", "watch"]
+
+  # external attacher
+- apiGroups: ["storage.k8s.io"]
+  resources: ["volumeattachments"]
+  verbs: ["get", "list", "watch", "update", "patch"]
+- apiGroups: ["storage.k8s.io"]
+  resources: ["volumeattachments/status"]
+  verbs: ["patch"]
+  # CSI nodes must be listed
+- apiGroups: ["storage.k8s.io"]
+  resources: ["csinodes"]
+  verbs: ["get", "list", "watch"]
+
+  # get kube-system namespace to retrieve Uid
+- apiGroups: [""]
+  resources: ["namespaces"]
+  verbs: ["get"]
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: mayastor-cluster-role-binding
+subjects:
+- kind: ServiceAccount
+  name: {{ .Release.Name }}-service-account
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: mayastor-cluster-role
+  apiGroup: rbac.authorization.k8s.io
+```
+
+{% endtab %}
+
+   
+
+## Minimum worker node count
+
+  The minimum supported worker node count is three nodes. When using the synchronous replication feature (N+1 mirroring), the number of worker nodes to which Mayastor is deployed should be no less than the desired replication factor.
```

Comment:
I think that should be "(N-way mirroring)"

Reply:
Updated it.